staging/src/k8s.io/client-go/tools/cache/shared_informer_test.go

```diff
@@ -182,74 +182,150 @@ func TestListenerResyncPeriods(t *testing.T) {
 	}
 }
 
-func TestResyncCheckPeriod(t *testing.T) {
+func TestResyncCheckPeriodWithResync(t *testing.T) {
+	checkResyncCheckPeriod(t, true)
+}
+
+func TestResyncCheckPeriodSansResync(t *testing.T) {
```

Comment:
The style of the moment is to do it like this:

```
func TestResyncCheckPeriod(t *testing.T) {
  t.Run("WithResync", func(t *testing.T) { checkResyncCheckPeriod(t, true) })
  t.Run("SansResync", func(t *testing.T) { checkResyncCheckPeriod(t, false) })
}
```

Reply:
revised to be like that